CHANGELOG.md

```diff
@@ -1,12 +1,19 @@
 # CHANGELOG
 
+## 7.0.0-beta.3.1
```

Comment:
let's bump this to `7.0.0-beta.4`, we should have bumped minors while migrating to nnbd, but I don't think we want to have two numbers after beta